x-pack/test/functional/apps/ml/short_tests/model_management/model_list.ts

```diff
@@ -157,6 +181,59 @@ export default function ({ getService }: FtrProviderContext) {
         await ml.securityUI.logout();
       });
 
+      describe('Add trained model flyout', () => {
+        before(async () => {
+          await ml.testExecution.logTestStep('Open the Add Trained Model Flyout');
+          await ml.trainedModelsFlyout.open();
```

Comment:
We can assert that flyout is open (or closed) inside of the `open` method. Similar to the other methods we have, e.g. https://github.com/elastic/kibana/blob/main/x-pack/test/functional/services/ml/trained_models_table.ts#L502

Reply:
and you can also have just one method for assertion, i.e. `assertOpen(expectOpen: boolean = true)`